### DISCUSSION TITLE
Help with "ValueError: unrecognized engine ee"

### DISCUSSION THREAD

**Discussion by francobarrionuevoenv21** [google/Xee#150] [Q&A]
Hi, Community!

I'm trying to retrieve MODIS MOD13Q1.061 Terra Vegetation Indices 16-Day Global 250m product from GEE, but after running the following code 

```
ds = xarray.open_dataset(
    withNdvi,
    engine='ee',
    crs='EPSG:4326',
    scale=250,
    geometry=featureCollection01.geometry(),
    ee_mask_value=-9999,
)
```

I'm getting this error:

`ValueError: unrecognized engine ee must be one of: ['h5netcdf', 'scipy', 'store']`

The thing is that I tried with other datasets and I got the same error. I've searching a little about why this problem arises, but I couldn't find anything helpful yet. 

Thank you in advance!

**Comment by tylere** [comment]
Can you confirm you imported the Xee library? This is what adds the `ee` engine option.
```import xee```

If that doesn't fix the issue, could you provide a Colab notebook that reproduces the error?
